### DISCUSSION TITLE
Can vaspGibbs be used to calculate Gibbs free energy in solution?

### DISCUSSION THREAD

**Discussion by leem258** [ftherrien/VaspGibbs#7] [VaspGibbs Forum]
Do you have suggestions for calculating the frequency and zero-point energy of a molecule in a vacuum and the total energy in water? That is, turn off LSOL when optimizing the structure and calculating the frequency, and turn on LSOL when calculating the total energy.

**Comment by ftherrien** [comment]
Hi @leem258,

If you want to get ZPE and frequencies, first run your optimization in VASP as usual, then, once the calculation is done, run a job with:

`vasp_gibbs --molecule`

If you run your job in parallel using `srun` for example it would look like this:

`vasp_gibbs --molecule -n 32 -m "srun" -v "path/to/vasp_std"`

You will get the ZPE in the VaspGibb.md file and the frequencies will be in the OUTCAR.

You can then run your LSOL calculation and use the resulting energy. If you want Gibbs free energy, you can use "G - E_dft" from the last line of VaspGibbs.md and add it to your new energy.

Does that help?
